src/index.js

```diff
@@ -0,0 +1,20 @@
+import './assets/css/style.css';
```

Comment:
Not sure why you are not using [`sass-loader`](https://github.com/webpack-contrib/sass-loader) with Webpack and directly use the `scss` file that you have 😕.